addons/l10n_tr_nilvera_einvoice/models/account_edi_xml_ubl_tr.py

```diff
@@ -53,6 +55,9 @@ def _get_country_vals(self, country):
     def _get_partner_party_identification_vals_list(self, partner):
         # EXTENDS account.edi.xml.ubl_21
         vals = super()._get_partner_party_identification_vals_list(partner)
+        # Nilvera will reject any <ID> without a <schemeID>, so remove all items not
+        # having the following structure : {'id': '...', 'id_attrs': {'schemeID': '...'}}
+        vals = list(filter(lambda v: ('id' in v and 'id_attrs' in v and 'schemeID' in v['id_attrs']), vals))
```

Comment:
This could also be done this way
```suggestion
        vals = [v for v in vals if v.get('id') and v.get('in_attrs', {}).get('schemeID')]
```